charts/seed-bootstrap/charts/elastic-kibana-curator/templates/kibana/kibana-network-policy.yaml

```
@@ -0,0 +1,30 @@
+apiVersion: {{ include "networkpolicyversion" . }}
+kind: NetworkPolicy
+metadata:
+  annotations:
+    gardener.cloud/description: |
+      Allows Ingress to the Kibana from the nginx controller in 'kube-system' namespace,
```

```suggestion
      Allows Ingress to the Kibana from the nginx controller in 'kube-system' namespace.
```
I think `allow-to-elasticsearch` handles the egress to elasticsearch.